Dockerfile.downstream

```diff
@@ -1,27 +1,31 @@
+ARG TARGETARCH
+ARG COMMIT
+
 # Build the manager binary
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23 as builder
+FROM --platform=linux/$TARGETARCH brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23 as builder
 
+ARG TARGETARCH=amd64
 ARG VERSION="unknown"
 ARG COMMIT
+ARG LDFLAGS
```

Comment:
hmm who set that argument I checked flp and operator they don't seem to use it ?

Reply:
The updstream Dockerfile for the ebpf agent is using one:
https://github.com/netobserv/netobserv-ebpf-agent/blob/cee90d9b0bc85eb72644b896711ea6637335ad61/Dockerfile#L7

This is to align with it.

Reply:
but its set in the Makefile I am not sure if konflux pipeline will set tbh ?

Reply:
if its not set then we need to set like in https://github.com/netobserv/netobserv-ebpf-agent/blob/main/Makefile#L29